feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -158,12 +159,12 @@ class PreviewViewModel @Inject constructor(
         }
     }
 
-    fun captureImage() {
+    fun captureImage(blinkState: BlinkState?) {
```

Comment:
I think this is breaking UDF. The UI should be reacting to state rather than passing it down to the ViewModel.

One way I can think to reverse this is to add a capture counter integer to PreviewUiState that gets incremented every time the onCaptureStarted callback is called. In the UI, we can have a LaunchedEffect keyed on this integer that runs the BlinkState animation.

This will also allow us to push BlinkState into  a `remember` variable inside the `PreviewDisplay` composable and we won't need to pass it around to other components.

We're doing something similar with the zoom UI here: https://github.com/google/jetpack-camera-app/blob/17c50b55f1dcd19bc2752a7d765a815895cc2558/feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/ui/CameraControlsOverlay.kt#L92